src/mongo/MongoScrapbook.ts

```diff
@@ -247,7 +247,17 @@ class FindMongoCommandsVisitor extends MongoVisitor<MongoCommand[]> {
 				const argAsObject = this.contextToObject(ctx);
 				const argText = EJSON.stringify(argAsObject);
 				lastCommand.arguments.push(argText);
-				lastCommand.argumentObjects.push(EJSON.parse(argText));
+				let removeDuplicatedBackslash = /\\{4}(?=[0-9bwds.*])/gi;
```

Comment:
Please remember to comment when you're doing some unexplained like this. Maintainers will have absolutely no clue why you're doing this. It also helps you if you have to explain it in comments.

Suggested comment (please check for correctness):

// In a regex, \b matches a word boundary: {$regex: /\b/}
// But with a string, \b means a backspace, so a word boundary equivalent using a string is {$regex: "\\b"}
// So converting from string to regex means we need to 

…  see, I can't even remember the full details from yesterday's discussion...  That's why we need comments. Can you please finish or modify this comment to explain why you're doing this?  Thx.

Reply:
Done in the next commit. Please let me know if it needs more explanation.